_templates/README.md.erb

```diff
@@ -1,47 +1,80 @@
+name: Example workflow for <%= @name %> using Snyk
 on: push
 jobs:
   security:
     runs-on: ubuntu-latest
     steps:
-    - uses: actions/checkout@master
-    - name: Run Snyk to check for vulnerabilities
-      uses: snyk/actions/<%= @variant.downcase %>@master
-      env:
-        SNYK_TOKEN: ${{ secrets.SNYK_TOKEN }}
+      - uses: actions/checkout@master
+      - name: Run Snyk to check for vulnerabilities
+        uses: snyk/actions/<%= @variant.downcase %>@master
+        env:
+          SNYK_TOKEN: ${{ secrets.SNYK_TOKEN }}
 ```
 
-The Snyk <%= @name %> Action has properties which are passed to the underlying image. These are
-passed to the action using `with`.
+## Properties
 
-| Property | Default | Description |
-| --- | --- | --- |
-| args |   | Override the default arguments to the Snyk image |
-| command | test | Specify which command to run, for instance test or monitor |
-| json | false | In addition to the stdout, save the results as snyk.json |
+The Snyk <%= @name %> Action has properties which are passed to the underlying image. These are passed to the action using `with`.
+
+| Property | Default | Description                                                                                         |
+| -------- | ------- | --------------------------------------------------------------------------------------------------- |
+| args     |         | Override the default arguments to the Snyk image. See [Snyk CLI reference for all options][cli-ref] |
+| command  | test    | Specify which command to run, for instance test or monitor                                          |
+| json     | false   | In addition to the stdout, save the results as snyk.json                                            |
 
 For example, you can choose to only report on high severity vulnerabilities.
 
 ```yaml
-name: Example workflow for <%= @name %> using Snyk 
+name: Example workflow for <%= @name %> using Snyk
+on: push
+jobs:
+  security:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@master
+      - name: Run Snyk to check for vulnerabilities
+        uses: snyk/actions/<%= @variant.downcase %>@master
+        env:
+          SNYK_TOKEN: ${{ secrets.SNYK_TOKEN }}
+        with:
+          args: --severity-threshold=high
+```
+
+## Uploading Snyk scan results to GitHub Code Scanning
+
+Using `--sarif-file-output` [Snyk CLI flag][cli-ref] and the [official GitHub SARIF upload action](https://docs.github.com/en/code-security/secure-coding/uploading-a-sarif-file-to-github), you can upload Snyk scan results to the GitHub Code Scanning.
+
+![Snyk results as a SARIF output uploaded to GitHub Code Scanning](../_templates/sarif-example.png)
+
+Snyk Action will fail when vulnerabilities are found. This would prevent SARIF upload action to run, so we need to introduce a [continue-on-error](https://docs.github.com/en/actions/reference/workflow-syntax-for-github-actions#jobsjob_idstepscontinue-on-error) option like so:
```

Comment:
```suggestion
The Snyk Action will fail when vulnerabilities are found. This would prevent the SARIF upload action from running, so we need to introduce a [continue-on-error](https://docs.github.com/en/actions/reference/workflow-syntax-for-github-actions#jobsjob_idstepscontinue-on-error) option like this:
```

Reply:
Looks like this change also needs to be made in 15 other places if you want to accept it.